backend/src/main/java/com/crm/dto/request/CustomerRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.crm.dto.request;
+
+import com.crm.exception.ErrorDict;
+
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
I tutaj też

Reply:
To nie zadziała z Builderem, zostawiłem jak było